Submodels/MTSOL_call.py

```
@@ -1,20 +1,621 @@
+        if exit_flag == ExitFlag.NON_CONVERGENCE.value:
+            self.HandleNonConvergence()
+        elif exit_flag == ExitFlag.CRASH.value:
+            self.CrashRecovery(case_type,
+                               iter_count,
+                               )
+        elif (exit_flag == ExitFlag.COMPLETED.value or 
+              exit_flag == ExitFlag.SUCCESS.value or 
+              exit_flag == ExitFlag.NOT_PERFORMED.value):
+            return
+        else:
+            raise OSError(f"Unknown exit flag {exit_flag} encountered!") from None
```

Ruff (0.8.2)

Avoid specifying long messages outside the exception class

(TRY003)